notebooks/responsibleaidashboard/responsibleaidashboard-census-classification-model-debugging.ipynb

```diff
@@ -257,6 +257,80 @@
     "rai_insights.compute()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "b84c6c0d",
+   "metadata": {},
+   "source": [
+    "Compose some cohorts which can be injected into the `ResponsibleAIDashboard`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0994b7d6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from raiwidgets.cohort import Cohort, CohortFilter, CohortFilterMethods\n",
+    "\n",
+    "# Cohort on continuos feature in the dataset\n",
+    "cohort_filter_continuous_1 = CohortFilter(\n",
+    "    method=CohortFilterMethods.METHOD_LESS,\n",
+    "    arg=[65],\n",
+    "    column='age')\n",
+    "cohort_filter_continuous_2 = CohortFilter(\n",
+    "    method=CohortFilterMethods.METHOD_GREATER,\n",
+    "    arg=[40],\n",
+    "    column='hours-per-week')\n",
+    "\n",
+    "user_cohort_continuous = Cohort(name='Cohort Continuous')\n",
+    "user_cohort_continuous.add_cohort_filter(cohort_filter_continuous_1)\n",
+    "user_cohort_continuous.add_cohort_filter(cohort_filter_continuous_2)\n",
+    "\n",
+    "# Cohort on categorical feature in the dataset\n",
```

Comment:
Did you mean "features" (plural)? In any case, it's not obvious to me why it's named like this. How about naming them after the filters you put, e.g., age between 40 and 65?

Reply:
Updated. Please check.